tests/decryption/test_decrypt.py

```diff
@@ -31,7 +36,72 @@ def test_get_keys(self, files, expected_keys_retrieved):
         Ensure that `get_private_keys` gets only private keys from a list of files.
         """
         def get_expected_bytes():
-            return [get_pk_bytes(INPUT_DIR/filename, callback=lambda x: '')
+            return [get_sk_bytes(INPUT_DIR/filename, callback=lambda x: '')
                     for filename in expected_keys_retrieved]
 
         assert get_private_keys([INPUT_DIR/file for file in files]) == get_expected_bytes()
+
+
+class TestDecryptFiles:
+    """Test decrypt_files."""
+
+    @pytest.fixture()
+    def key_pair(self):
+        """Returns the key pair used to encrypt the input files."""
+        return INPUT_DIR/"alice.sec", INPUT_DIR/"alice.pub"
+
+    @pytest.fixture()
+    def key_pair_bytes(self, key_pair):
+        """Returns the bytes of the key pair."""
+        sk, pk = key_pair
+        sk_bytes = get_sk_bytes(filepath=sk, callback=lambda x: '')
+        pk_bytes = get_pk_bytes(filepath=pk)
+        return sk_bytes, pk_bytes
+
+    @pytest.fixture()
+    def encrypted_files(self, key_pair_bytes):
+        """Returns the encrypted file paths and re-encrypts files after use."""
```

Comment:
Maybe I'm misunderstanding the use-case here, but I think it is a bit brittle to manually re-encrypt the files after each test. Instead, I would recommend copying over the test files to a random directory prior to the test, and removing that directory at the end of the test. Pytest has some fixtures to help with this: https://docs.pytest.org/en/6.2.x/tmpdir.html.